azure-pipelines.yml

```diff
@@ -59,10 +59,14 @@ stages:
               imageName: 'windows-2019'
               rustup_toolchain: stable
               target: 'x86_64-pc-windows-msvc'
-            mac-stable:
+            mac-stable-intel:
               imageName: 'macos-11'
               rustup_toolchain: stable
               target: 'x86_64-apple-darwin'
+            mac-stable-arm:
+              imageName: 'macos-11'
+              rustup_toolchain: stable
+              target: 'aarch64-apple-darwin'
```

Comment:
I can't find it in the docs, can you point to where it's shown to be available?

Reply:
I meant, where is it available in azure pipelines. I don't see ARM in in https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#software

Would cross-compilation work out of the box? I don't have an Intel mac to test

Reply:
I’m pretty sure it will work out of the box. Can test this weekend.

Reply:
Yes, cross-compilation works out of the box. 
Below I use the same commands as used in the azure pipelines. 
This is run on  an intel Mac and produces an arm executable.
 The Zola sources are in `/Volumes/Data3/zola`

(The rust tool chain is really great!)

```
zola@mac-mini ~ % uname -a
Darwin mac-mini 21.6.0 Darwin Kernel Version 21.6.0: Fri Sep 15 16:17:23 PDT 2023; root:xnu-8020.240.18.703.5~1/RELEASE_X86_64 x86_64
zola@mac-mini ~ % curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable
info: downloading installer
... download info omitted ...
info: default toolchain set to 'stable-x86_64-apple-darwin'

  stable-x86_64-apple-darwin installed - rustc 1.73.0 (cc66ad468 2023-10-03)


Rust is installed now. Great!

To get started you may need to restart your current shell.
This would reload your PATH environment variable to include
Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, run:
source "$HOME/.cargo/env"
zola@mac-mini ~ % source $HOME/.cargo/env 
zola@mac-mini ~ % rustup target add aarch64-apple-darwin
info: downloading component 'rust-std' for 'aarch64-apple-darwin'
info: installing component 'rust-std' for 'aarch64-apple-darwin'
 24.5 MiB /  24.5 MiB (100 %)  10.1 MiB/s in  2s ETA:  0s
zola@mac-mini ~ % cd /Volumes/Data3/zola 
zola@mac-mini zola % cargo build --release --target aarch64-apple-darwin
    Updating crates.io index
  Downloaded unicode-ident v1.0.8
... download and compilation lines omitted ...
    Finished release [optimized] target(s) in 14m 23s
zola@mac-mini zola % file target/aarch64-apple-darwin/release/zola
target/aarch64-apple-darwin/release/zola: Mach-O 64-bit executable arm64
```

Reply:
I thought there would be some issues with reqwest and the tls features but i guess it's ok!